tests/sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/telemetry.clients_daily.schema.json

```diff
@@ -597,6 +597,23 @@
     "name": "search_content_urlbar_searchmode_sum",
     "type": "RECORD"
   },
+  {
+    "fields": [
+      {
+        "mode": "NULLABLE",
+        "name": "key",
+        "type": "STRING"
+      },
+      {
+        "mode": "NULLABLE",
+        "name": "value",
+        "type": "INTEGER"
+      }
+    ],
+    "mode": "REPEATED",
+    "name": "search_content_urlbar_persisted_sum",
+    "type": "RECORD"
+  },
```

Comment:
There should be schema entries here for `search_adclicks_urlbar_persisted_sum` and `search_withads_urlbar_persisted_sum` as well